README.md

```diff
@@ -230,6 +255,13 @@ export function useLoading() {
 }
 ```
 
+A helper function that automatically types tuples can also be helpful if you write a lot of custom hooks:
+```ts
+function tuplify<T extends any[]>(...elements: T) { return elements }
+
+const myTuple = tuplify(false, 1, 'a') // type is [boolean, number, string]
```

Comment:
I would display actual usage here. I guess this is used to defeat type widening for tuples? 
```ts
function useMixedArray() {
  return [42, 'Hello, World!']; // $ExpectType (number | string)[]
}

function useTuple() {
  return tuplify([42, 'Hello, World!']); // $ExpectType [number, string]
}
```

Maybe go as far as recommend explicit return types which should also solve type widening issues with other literal types.

Reply:
I'm fine with the current text. It comes down to personal preference I suppose.